mail/views_test.py

```diff
@@ -199,6 +199,53 @@ def test_course_team_email_unpaid(self):
         assert resp.status_code == status.HTTP_403_FORBIDDEN
 
 
+class LearnerMailViewTests(APITestCase, MockedESTestCase):
+    """
+    Tests for LearnerMailView
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        with mute_signals(post_save):
+            staff_profile = ProfileFactory.create(user__email='sender@example.com')
+            recipient_profile = ProfileFactory.create(user__email='recipient@example.com', student_id=123)
+        cls.staff_user = staff_profile.user
+        cls.recipient_user = recipient_profile.user
+        Role.objects.create(
+            user=cls.staff_user,
+            program=ProgramFactory.create(),
+            role=Staff.ROLE_ID
+        )
+        cls.url_name = 'learner_mail_api'
+        cls.request_data = {
+            'email_subject': 'email subject',
+            'email_body': 'email body'
+        }
+
+    @patch('mail.views.MailgunClient')
+    def test_send_learner_email_view(self, mock_mailgun_client):
+        """
+        Test that learner emails are correctly sent through the view
+        """
+        self.client.force_login(self.staff_user)
+        mock_mailgun_client.send_individual_email.return_value = Mock(
+            spec=Response,
+            status_code=status.HTTP_200_OK,
```

Comment:
we don't currently have tests that simulate a mailgun failure for any of the other email views. it will involve patching `requests.post` or something like that. i made a quick attempt at it and failed. it's non-trivial and i would very strongly prefer creating a separate issue for that work. if you have any objections let me know